src/main/java/sg/ncl/MainController.java

```diff
@@ -144,39 +144,121 @@ public String tools() {
         return "tools";
     }
 
-    @RequestMapping(value="/futureplan/download", method=RequestMethod.GET)
-    public void futureplanDownload(HttpServletResponse response) throws FuturePlanDownloadException, IOException {
+    @RequestMapping("/createaccount")
+    public String createaccount() {
+        return "createaccount";
+    }
+
+    @RequestMapping("/createexperiment")
+    public String createexperimenttutorial() {
+        return "createexperiment";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)